### DISCUSSION TITLE
Sparkasse: BLZ vs BIC

### DISCUSSION THREAD

**Discussion by arunoruto** [firefly-iii/firefly-iii#5428] [Firefly III]
I have a bank account at the Sparkasse and I am trying to import the data via a CSV file.
There is a peculiar problem, where one column is dedicated for the BIC of the recipient of the transfer, but if I make a withdrawal the value is the BLZ (not sure if of my account or the ATM of the bank, I never withdraw money outside my city). Is there a way of mapping a value to another, in this case to map the BLZ to a BIC. Or at least skip the insertion of the BIC for the case of a withdrawal of the money. For the later solution, it would be needed to map the values of withdrawal and transfer, i.e., :
```json
{
    "withdrawal": "BARGELDAUSZAHLUNG",
    "transaction": "FOLGELASTSCHRIFT"
}
```
or even make an array of values for a specific column.
It is just an idea! I will probably ignore the column of the BIC value and take care of it later.

**Comment by JC5** [comment]
There's not much I can do from the side of Firefly III, I'm afraid :(
